Frontteam branch

mike3104 opened the pull request:
コードを改善して、Qiitaにより近づけましたので、これをmasterとしたいです。
レビューよろしくお願いいたします。

Commit:
新着順（index.phpの元）のhtml・cssの大枠ができました。

新着順（index.phpの元）のhtml・cssができました。
よりQiitaに近づけたコードに改善しております。
かくリーダーにレビューいただいた後、問題なさそうであれば、あやかちんはこれをプルして、使用してください。

今後はこれにFigを入れ込む予定です。

kakudaisuke commented on index.html:
```diff
@@ -5,66 +5,226 @@
             <div class="header-start">
-                <div class="header-sitoname">
+                <div class="header-siteName">
                     サイト名
```
とりあえず仮のロゴを入れてもらえますか？

kakudaisuke commented on index.html:
```diff
@@ -5,66 +5,226 @@
                 </div>
                 <from class="header-search" action="cgi-bin/example.cgi" method="post">
                     <input class="header-search-btn1" type="submit" name="submit" value="検索">
                     <input class="header-search-btn2" type="search" name="search" placeholder="キーワードを入力してください。">
```
ブラウザだとplaceholderの文章が途中で切れてしまうので、改善をお願いします。

kakudaisuke commented on index.html:
```diff
@@ -5,66 +5,226 @@
+            </div>
+        </div>
+    </div>
     <footer>footer</footer>
```
qiitaみたいに、footerに
- [ ] ロゴを入れる
- [ ] メニューを載せる（運営者情報、規約、などなど）
を入れるというのはいかがでしょうか？？

kakudaisuke reviewed (commented):
- [ ] 仮の線を消して、仮の色を入れたデザインにしておくのは後にしますか？（CSS直せば全ページ変わるなら、後回しでもいいかもですね）

Commit:
HTML

ghost commented:
かくリーダーのコメントを反映し、
１、LIKE、出演日などの追加
２、ロゴの追加
３、footerのコンテンツの追加
を実現いたしました。かなりホームページらしくなりました。

Commit:
Add files via upload

Commit:
Merge pull request #11 from WebYorozuya/mike3104-patch-1

imgをフォルダに入れた状態でpushしました。

kakudaisuke commented:
プルリクされてないので、いわゆる機能上の「レビュー」ってのができない模様なので、次からプルリクお願いします！その方がレビューしやすいので。**まず、めっちゃサイトがかわいくなってきましたね、ナイス！**
- [ ] フッターの右側は「お問い合わせ」と「利用規約」だけにしてもらえますか？（僕がレビューコメントで「などなど」という不明確な指示を出したのがいけなかったですね...）
- [ ] あと、SNSも確かに僕も一瞬思ったのですが、SNSをやれる気がしないので、一旦削除しておいた方が良いかな？？
- [ ] `<div class="heart-number">16</div>` のクラス名は`like`とか`like-counter`とか`heart-counter`あたりでどうでしょう？
左の`<nav>`に`<a></a>`を入れてください。
- [ ] ページの上下左右に少しだけだぶつきがあるのですが、原因はユーザーエージェントスタイルと思われます。body要素のに`margin: 0; `を加えください。もしくはユーザーエージェントスタイルを無効化するのが一番いいと思うので、こちらのサイトを参照にやってみてください。ressがいいかも。[](https://memowo.com/study/codememo/user-agent-stylesheet/)

ghost commented:
プルリクされてないので、いわゆる機能上の「レビュー」ってのができない模様なので、次からプルリクお願いします！
→そういうことなんですね。ということはこのプルリクはクローズさせるということになり、修正したコードで新しいプルリクを立ち上げる。その際は｢Mike3104 index.php ２｣という名前をつけるということになるのでしょうか？

kakudaisuke commented:
ミケさん、ごめんなさい。確かにおっしゃる通りですね。。そうか、新しいプルリクをクローズしないといけないんですね。
いや、僕が間違えていたかもです。すみません、もうちょっと勉強してみます！

Commit:
修正変更

変更しました。
・headerの配色範囲の修正
・<link rel="stylesheet" href="https://unpkg.com/ress/dist/ress.min.css">の追加
→理解はできていませんが、とりいそぎ修正しました。
・aタグの追加
・フッターのコンテンツ　「お問い合わせ」と「利用規約」だけ
・SNSの削除
・like-counterへの修正

kakudaisuke commented on index.html:
```diff
@@ -13,13 +14,13 @@
         <div class="header-container">
             <div class="header-start">
                 <div class="header-siteName">
-                    <img src="images/JazzLog-logo-black.png" class="siteLogo">
```
backgroundにカラー次第で、黒か白か選択お願いします（配色決まってからでもいいですが）

kakudaisuke commented on index.html:
```diff
@@ -35,26 +36,27 @@
+                <ul class="pc-nav">
                     <li class="li-newInformation">
-                        <img src="images/icons8-music-24-black.png" class="li-newInformationImg">  
+                        <a href=""><img src="images/icons8-music-24-black.png" class="li-newInformationImg"> </a>  
```
navの中のこのaタグ、文字に反映されてないですね。あと</a>の前の半角スペースも除去しておいた方がいいかもです。

kakudaisuke commented on index.html:
```diff
@@ -291,37 +234,30 @@
-                    利用規約</br>
-                    ガイドライン</br>
-                    ヘルプ</br>
+                <a href=""><div class="footer-right-information1"></a>
```
この辺のaタグも効いてないです！

kakudaisuke reviewed (commented):
ありがとうございます。
この辺をしっかり修正できたら、どんどん他のページをやってきましょうかね！

Commit:
HTML＆CSS　修正

aタグ修正等
※ロゴの選択は未実施

Commit:
テスト

テスト

Commit:
branch

Commit:
Merge pull request #14 from WebYorozuya/mike3104-patch-2

Mike3104 patch 2

Commit:
アカウント作成のhtmlです

Commit:
アカウント作成のCSSです

Commit:
配色入れてあります

Commit:
Merge pull request #20 from WebYorozuya/ayakaaa182-patch-2

style.cssに配色追加

kakudaisuke commented on index.html:
```diff
@@ -3,72 +3,265 @@
-                <div class="header-sitoname">
-                    サイト名
+                <div class="header-siteName">
+                    <a href=""><img src="images/JazzLog-logo-black.png" class="siteLogo"></a>
```
この背景色だとロゴは白の方が良さそうですね？footerも然りですが。配色一旦仮決めしたと思うので、白に変更お願いします。

mike3104 replied:
今マージによる影響の実験中なので、まだ内容を確認していません。

kakudaisuke commented on index.html:
```diff
@@ -3,72 +3,265 @@
+            <nav>
+                <ul class="pc-nav">
+                    <li class="li-newInformation">
+                        <a href=""><img src="images/icons8-music-24-black.png" class="li-newInformationImg"></a>
```
- [ ] このaタグは一つにまとめられるかと思います。href内は、まだ入れるものがなければ`#`を入れておきましょう。
- [ ] リンクの下線は消したいところですね。確かtext-decoration: none;でしたっけ？（他の箇所のaタグも全てです）

kakudaisuke commented on index.html:
```diff
@@ -3,72 +3,265 @@
+                </div>
+            </div>
+            <div class="footer-right">
+                <a href=""><div class="footer-right-information1">
```
footer内のメニューの配置が隣合わさってしまって見づらいのと、フォントの色も視認性がよくないので改善をお願いします。

kakudaisuke commented on index.html:
```diff
@@ -3,72 +3,265 @@
                 </div>
             </div>
         </div>
+        <div class="search-box"><input class="header-search-text2" type="search" name="search" placeholder="キーワードで検索"></div>
```
おそらくこれが変なところに表示されてしまっていて、ページ内に２つの「キーワードで検索」が見えてしまっています。こちらがレスポンシブで隠しておく要素ですかね？？

mike3104 replied:
Qiitaでは画面が小さくなると別のところに検索窓？が移動しています。
それを反映させたものです。

kakudaisuke commented on signup.html:
```diff
@@ -0,0 +1,61 @@
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="signup.css">
```
index.htmlを真似して、ユーザーエージェントスタイルシート（ブラウザのデフォルトのスタイル）をリセットしてください。

kakudaisuke commented on signup.html:
```diff
@@ -0,0 +1,61 @@
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="signup.css">
+    <title> 新規登録 </title>
```
通常タグと要素の間にスペースは入れないかと思います。`<title>新規登録</title>`とすべきではないでしょうか。ページ内全て確認をお願いします。

kakudaisuke commented on signup.html:
```diff
@@ -0,0 +1,61 @@
+         　　<input type="hidden" name="hidden" value="hidden">
+        </div>
+        <div class="signup_form"> 
+            <label> メールアドレス <input class="btn_signup" type="email" name="email" required></label>
```
アカウント名の入力フォームは左詰め、メアドとPWは中央揃えになっていますが、左詰めに統一した方が良さそうですね。

kakudaisuke commented on signup.html:
```diff
@@ -0,0 +1,61 @@
+         <p> ♪ 新規登録してね ♪ </p>
+    </div>
+ 
+    <div class="signup">
```
このdivを画面中央揃えにしてください。

kakudaisuke reviewed (changes requested):
みけさん、ayakaちんおつかれさまです！どんどん形になってきてますね(*^^*)
お二人はプルリク前にブラウザでどのように表示するか確認していますか？もししていなかったら、今後やってみてください。

kakudaisuke commented:
ブランチ構造の変更に伴い、こちらのプルリク変更もしくは一旦クローズかご対応お願いします。

ghost commented:
ブランチ改造に伴いクローズ
